### PR TITLE
Allow privileged containers

### PR DESCRIPTION
**Pull request by montaro**
See also #9546

### CODE REVIEW

File: installation/resources/installer-cr-cluster.yaml.tpl [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
 spec:
```
Comment (montaro) [inline installation/resources/installer-cr-cluster.yaml.tpl:9]: This file is copied from the master branch, I don't know why it's included here!

**Review by marcobebway — APPROVED** [review]

**Pull request merged**